### DISCUSSION TITLE
Tabele Builder: Remove the "Create record" button on the list-page

### DISCUSSION THREAD

**Discussion by wpas81** [filamentphp/filament#9419] [Help] [answered]
Hi

I've got an OrderResource page which lists all orders. The creation happens in another part of the app. So, while the users still has the rights to create an order,  I want to remove the create-button on the top resource page.

As far as I can find it seems to that this requires a custom layout but surely this must be toggleable via an option?

**Comment by wpas81** [comment, marked as answer]
Lol... obviously I find this right after posting it here:

ListOrders.php

```
protected function getHeaderActions(): array
    {
        return [
            //Actions\CreateAction::make(),
        ];

    }
```

**Reply by leandrocfe** [reply]
Close the topic please 👍
